activity.js

```diff
@@ -2,35 +2,34 @@ document.addEventListener('DOMContentLoaded', function () {
 
   chrome.storage.sync.get(["contrast_service_key", "teamserver_url"], function (items) {
 
-
     if (items["contrast_service_key"] != null) {
 
       getActivities(function () {
         return function (e) {
-          xhr = e.currentTarget
+          xhr = e.currentTarget;
           if (xhr.readyState == 4) {
             if (xhr.status == 403) {
               //Configuration problem
-              $("body").addClass("configuration-problem")
-              $("#configuration-problem").show()
+              $("body").addClass("configuration-problem");
+              $("#configuration-problem").show();
             } else if (xhr.responseText == "") {
-              $("body").addClass("no-activity")
-              $("#get-started").show()
+              $("body").addClass("no-activity");
+              $("#get-started").show();
             } else {
               json = JSON.parse(xhr.responseText)
-              console.log(json["activities"])
+              console.log(json["activities"]);
 
-              activities = json["activities"]
+              activities = json["activities"];
 
               if (activities === "undefined" || activities.length == 0) {
```

Comment:
Shouldn't this be **undefined** without the **"** ?

Reply:
Fixed